cmd/crio/main.go

```diff
@@ -325,8 +325,8 @@ func main() {
 			Value:  libkpod.DefaultHooksDirPath,
 			Hidden: true,
 		},
-		cli.StringFlag{
-			Name:   "default-mounts-path",
+		cli.StringSliceFlag{
+			Name:   "default-mounts",
 			Usage:  "set the default mounts file path",
```

Comment:
This isn't a file path anymore but actual mounts host:container.

Reply:
right, fixed.